prospero-cli/src/main/java/org/wildfly/prospero/cli/CliMain.java

```diff
@@ -72,12 +76,18 @@ public static CommandLine createCommandLine(Console console, ActionFactory actio
         commandLine.addSubcommand(new HistoryCommand(console, actionFactory));
         commandLine.addSubcommand(new RevertCommand(console, actionFactory));
         commandLine.addSubcommand(new RepositoryCommand(console, actionFactory));
+        commandLine.addSubcommand(new ChannelCommand(console, actionFactory));
 
         CommandLine repoCmd = commandLine.getSubcommands().get(CliConstants.Commands.REPOSITORY);
         repoCmd.addSubcommand(new RepositoryAddCommand(console, actionFactory));
         repoCmd.addSubcommand(new RepositoryRemoveCommand(console, actionFactory));
         repoCmd.addSubcommand(new RepositoryListCommand(console, actionFactory));
 
+        CommandLine channelCmd = commandLine.getSubcommands().get(CliConstants.Commands.CHANNEL);
+        channelCmd.addSubcommand(new ChannelAddCommand(console, actionFactory));
```

Comment:
With those small subcommands, do you think it would be tidier if they were inner classes of `ChannelCommand`? Or maybe moved to a separate package per sub-channel?

Reply:
I think both ways would work, they don't contain very much logic... Will update.